momento-sdk/src/intTest/java/momento/sdk/SimpleCacheClientTest.java

```diff
@@ -64,25 +61,30 @@ void teardown() {
 
   @Test
   public void createCacheGetSetDeleteValuesAndDeleteCache() {
-    String cacheName = UUID.randomUUID().toString();
-    String key = UUID.randomUUID().toString();
-    String value = UUID.randomUUID().toString();
+    final String cacheName = UUID.randomUUID().toString();
+    final String alternateCacheName = UUID.randomUUID().toString();
+    final String key = UUID.randomUUID().toString();
+    final String value = UUID.randomUUID().toString();
 
     target.createCache(cacheName);
-    CacheSetResponse response = target.set(cacheName, key, value);
-
-    CacheGetResponse getResponse = target.get(cacheName, key);
-    assertEquals(CacheGetStatus.HIT, getResponse.status());
-    assertEquals(value, getResponse.string().get());
+    target.createCache(alternateCacheName);
+    try {
+      target.set(cacheName, key, value);
 
-    target.delete(cacheName, key);
-    CacheGetResponse getAfterDeleteResponse = target.get(cacheName, key);
-    assertEquals(CacheGetStatus.MISS, getAfterDeleteResponse.status());
+      final CacheGetResponse getResponse = target.get(cacheName, key);
+      assertThat(getResponse).isInstanceOf(CacheGetResponse.Hit.class);
+      assertThat(((CacheGetResponse.Hit) getResponse).valueString()).isEqualTo(value);
 
-    CacheGetResponse getForKeyInSomeOtherCache = target.get(System.getenv("TEST_CACHE_NAME"), key);
-    assertEquals(CacheGetStatus.MISS, getForKeyInSomeOtherCache.status());
+      target.delete(cacheName, key);
+      final CacheGetResponse getAfterDeleteResponse = target.get(cacheName, key);
+      assertThat(getAfterDeleteResponse).isInstanceOf(CacheGetResponse.Miss.class);
```

Comment:
In other test suites, we include a string explaining what the value was when the assertion fails. Will the test runner output this by default?

Reply:
There will be no description by default other than saying the type is wrong. I could add descriptions.